docs/en/integrations/dbt/dbt-connecting.md

```diff
@@ -0,0 +1,96 @@
+---
+sidebar_label: Connecting to ClickHouse
+sidebar_position: 3
+description: Connecting dbt to ClickHouse
+---
+
+# Connecting ClickHouse
+
+1. Create a dbt project. In this case we name this after our imdb source. When prompted, select `clickhouse` as the database source.
+
+    ```bash
+    clickhouse-user@clickhouse:~$ dbt init imdb
+
+    16:52:40  Running with dbt=1.0.4
+    Which database would you like to use?
+    [1] clickhouse
+
+    (Don't see the one you want? https://docs.getdbt.com/docs/available-adapters)
+
+    Enter a number: 1  
+    16:53:21  No sample profile found for clickhouse.
+    16:53:21  
+    Your new dbt project "imdb" was created!
+
+    For more information on how to configure the profiles.yml file,
+    please consult the dbt documentation here:
+
+    https://docs.getdbt.com/docs/configure-your-profile
+    ```
+
+2. `cd` into your project folder.
+
+    ```bash
+    cd imdb
+    ```
+
+3. At this point, you will need the text editor of your choice. In the examples below, we use the popular VSCode. Opening the IMDB directory, you should see a collection of yml and sql files.
```

Comment:
For some reason, the screenshot images in this section are blurred and low quality. It's better to replace them with code blocks. For the folder structure, you can use the output of a `tree` command for eg.

Reply:
I'd rather use screenshots, they are visually easier to consume imo. Will bump resolution

Reply:
bumped the resolution